aiohttp/client_reqrep.py

```diff
@@ -475,7 +477,14 @@ def keep_alive(self):
             if self.url.raw_query_string:
                 path += '?' + self.url.raw_query_string
 
-        writer = StreamWriter(conn.protocol, conn.transport, self.loop)
+        async def on_chunk_sent(chunk):
```

Comment:
Convert the function into ClientRequest's method. No need to create a nested function on every HTTP request.

Reply:
done